uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-balance-tab.component.js

```diff
@@ -22,16 +22,18 @@ define([
   function LeaveBalanceTabController ($q, $scope, AbsencePeriod, AbsenceType,
     LeaveBalanceReport, notification, Session) {
     var filters = {};
-    var pageSize = 50;
     var vm = this;
 
     vm.absencePeriods = [];
     vm.absenceTypes = [];
     vm.loading = { component: true, report: true };
     vm.loggedInContactId = null;
+    vm.pagination = { page: 1, size: 50 };
```

Comment:
I made a comment, on the screenshot I'm only displaying 10 records to showcase pagination ( because I only have 26 records available :'( )